java/client/build.gradle

```diff
@@ -31,8 +31,29 @@ dependencies {
     testImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '3.12.4'
 }
 
+ext {
+    checkProtocVersion = { String output ->
+        // Line in format like: libprotoc 26.1
+        int majorVersion = Integer.parseInt(output.split(" ")[1].split("\\.")[0].trim());
+        int minorVersion = Integer.parseInt(output.split(" ")[1].split("\\.")[1].trim());
+        if (majorVersion < 26 || (majorVersion == 26 && minorVersion < 1)) {
+            throw new GradleException("Use Protoc version 26.1 or later");
+        }
+        return output.split(" ")[1]
+    }
+}
+
 tasks.register('protobuf', Exec) {
     doFirst {
+        new ByteArrayOutputStream().withStream { os ->
+            exec {
```

Comment:
How do you handle errors? If the user don't have protoc installed at all, lets catch it and raise an appropriate message with the required protobuf version